core/auth/src/main/java/software/amazon/awssdk/auth/credentials/ProcessCredentialsProvider.java

```diff
@@ -83,9 +92,11 @@ private ProcessCredentialsProvider(Builder builder) {
 
         cmd.add(builderCommand);
 
-        this.command = Collections.unmodifiableList(cmd);
+        this.executableCommand = Collections.unmodifiableList(cmd);
```

Comment:
why was this renamed

Reply:
Mostly to differentiate it from the `commandFromBuilder`.